app/tests.py

```diff
@@ -20,16 +22,17 @@
     INSTALLMENT_CONDITIONS,
     STATUS,
 )
-from app.views import (
-    download_csv,
-    InstallmentsFilter,
-    InstallmentsTableView,
-)
-from app.models import (
+from .models import (
```

Comment:
Is there a reason your changing absolute imports to relative imports? We tend to use absolute imports

Reply:
My bad, I will make them absolute :)